packages/amplify-provider-awscloudformation/src/print-cdk-migration-warning.ts

```diff
@@ -65,7 +65,7 @@ const getOverridesWarning = (resourcesToBuild: IAmplifyResource[], dependencyToS
 const getCustomResourcesWarning = (resourcesToBuild: IAmplifyResource[], dependencyToSearch: string): AmplifyWarning | undefined => {
   let customResourcesWarningObject;
   const customResourceImpactedFiles = [];
-  const customCategoryResources = resourcesToBuild.filter((resource) => resource.category === AmplifyCategories.CUSTOM);
+  const customCategoryResources = resourcesToBuild.filter((resource) => resource.category === AmplifyCategories.CUSTOM && resource.service !== 'customCloudformation');
```

Comment:
consider extracting second part of condition to `isCFNBasedCustomeResource(resource)` for future readers.